lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -171,84 +303,37 @@ class ChatInfoView extends StatelessWidget {
         members.insert(0, me);
       }
 
-      final style = Theme.of(context).style;
-
-      Widget bigButton({
-        Key? key,
-        Widget? leading,
-        required Widget title,
-        void Function()? onTap,
-      }) {
-        return SizedBox(
-          key: key,
-          height: 56,
-          child: Container(
-            decoration: BoxDecoration(
-              borderRadius: style.cardRadius,
-              border: style.cardBorder,
-              color: style.colors.transparent,
-            ),
-            child: Material(
-              type: MaterialType.card,
-              borderRadius: style.cardRadius,
-              color: style.cardColor.darken(0.05),
-              child: InkWell(
-                borderRadius: style.cardRadius,
-                onTap: onTap,
-                hoverColor: style.cardColor.darken(0.08),
-                child: Padding(
-                  padding: const EdgeInsets.fromLTRB(12, 12, 12, 12),
-                  child: Row(
-                    children: [
-                      const SizedBox(width: 4),
-                      Expanded(
-                        child: DefaultTextStyle(
-                          overflow: TextOverflow.ellipsis,
-                          maxLines: 1,
-                          style: style.fonts.normal.regular.primary,
-                          child: title,
-                        ),
-                      ),
-                      if (leading != null) ...[
-                        const SizedBox(width: 12),
-                        leading,
-                        const SizedBox(width: 4),
-                      ],
-                    ],
-                  ),
-                ),
-              ),
-            ),
-          ),
-        );
-      }
-
       return Column(
         mainAxisSize: MainAxisSize.min,
         children: [
-          bigButton(
-            key: const Key('AddMemberButton'),
-            leading: const SvgIcon(SvgIcons.addUser),
-            title: Text('btn_add_member'.l10n),
-            onTap: () =>
-                AddChatMemberView.show(context, chatId: c.chat?.id ?? id),
-          ),
-          const SizedBox(height: 3),
           ...members.map((e) {
             final bool inCall = c.chat?.chat.value.ongoingCall?.members
                     .any((u) => u.user.id == e.id) ==
                 true;
 
             return MemberTile(
               user: e,
-              canLeave: e.id == c.me,
-              inCall: e.id == c.me || c.chat?.chat.value.ongoingCall == null
+              myUser: e.id == c.me,
```

Comment:
```suggestion
              me: e.id == c.me,
```

`MyUser` во всём коде ассоциируется с моделькой, а не с флажком/айди.